gulpfile.babel.js

```diff
@@ -208,6 +208,14 @@ gulp.task('default', ['clean'], () => {
   );
 });
 
+//Generates new JWT
+gulp.task('createNewToken',[],() => {
+  let jwtSecret = config.jwtSecret;
+  let jwt = require('jsonwebtoken');
+  let token = jwt.sign({data : {name: 'foo',password: 'bar'}},jwtSecret , { expiresIn: '1h' });
```

Comment:
Can we set foo/bar in constants in another file, from the environment, or from the command line?